utils/transporter.js

```diff
@@ -0,0 +1,13 @@
+const nodemailer = require('nodemailer')
+
+const transporter = nodemailer.createTransport({
+  service: "gmail",
+  auth: {
+    user: "santanrathore75209@gmail.com",
```

Comment:
Better to use any credentials used in config files or default.json. Can be used at multiple places and will have to update at one place only, if needed.

Reply:
Okay

Reply:
ok